django/template/backends/django.py

```diff
@@ -62,6 +63,9 @@ def origin(self):
         return self.template.origin
 
     def render(self, context=None, request=None):
+        return mark_safe(''.join(self.stream(context, request)))
+
+    def stream(self, context=None, request=None):
         # A deprecation path is required here to cover the following usage:
```

Comment:
This `stream()` method is called from other `stream()` methods which get their arguments from `render()` methods. Moving this code from here would mean moving it to every other `render()` methods.